1 table sales.sql

```diff
@@ -0,0 +1,42 @@
+use BikeShopDB
+go
+drop table if exists Sales
+go
+create table dbo.Sales(
+	SaleId int not null identity primary key,
+	CustomerName varchar (30) not null
+		constraint c_sales_customer_name_cannot_be_blank check(CustomerName > ''),   
+	CustomerAddress varchar (70) not null, 

```

Comment:
Address should be 4 columns: address, city, state and zip. It's not good practice to store them all in one column since then reports can get much more complicated if you want to group by a certain part of the address...

Reply:
See 2 comments for state and zip